internal/policy/policy_test.go

```diff
@@ -337,3 +336,95 @@ func TestGetStateForCommit(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, firstState, state)
 }
+
+func TestFindDelegationEntry(t *testing.T) {
+	t.Run("no delegation", func(t *testing.T) {
+		state := createTestStateWithPolicy(t)
+
+		entry, err := state.findDelegationEntry("random")
+		assert.ErrorIs(t, err, ErrDelegationNotFound)
+		assert.Equal(t, entry, tuf.Delegation{})
```

Comment:
Hmm, here's what it returns for an error:

```
	if err != nil {
		return tuf.Delegation{}, err
	}
```

I think it's okay to sneak a change in to turn all of those returns into nil first, before the tests commit here. Could you?

Reply:
I have turned them into nil's, but to do that, the return value had to be changed into a pointer